### DISCUSSION TITLE
Strange CTF fitting results

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#60] [General]
I completed pre-processing.  Looking at CTFs, the were all fine, except two where the defocus was out of the search range.  So I increased the range and ran the pre-processing block again. But now several of the CTF fits that were fine before are completely messed up. In some cases the power spectrum looks ok, but the 1D profile is strange. Estimated resolution reported is very low.

<img width="899" alt="Screenshot 2024-02-05 at 23 10 03" src="https://github.com/nextpyp/.github/assets/20352352/62906ad1-f9b5-40e3-bd41-fe2ea12b8cae">

<img width="538" alt="Screenshot 2024-02-05 at 23 10 12" src="https://github.com/nextpyp/.github/assets/20352352/42c03b21-2664-4cde-bca5-c54d2686dd63">

In some cases the power spectrum image is also messed up. 

<img width="536" alt="Screenshot 2024-02-05 at 23 10 31" src="https://github.com/nextpyp/.github/assets/20352352/0918d191-24b5-4abe-8a14-40f045cb5111">

This has now happed several times and I hope I don't have to start the workflow again from beginning.

Edit: Changing the CTF range parameters back to what they were before, and rerunning the block does not restore the correct results I had originally. 

Edit2: Another thing that is different this time is that after the first successful pre-processing run I have deselected some tilts (tilt series->Tilts->show all tilts) and also made a set of good tilt series (using a filter and manual selection).

**Comment by abartesaghi** [comment]
None of this should happen. It is hard to tell from the images alone what may be going on. Do the logs show any errors or warnings?

**Comment by JuhaHuiskonen** [comment]
I haven't noticed any warnings or errors in the logs.  Perhaps you could try try repeat the error by running pre-processing, then change CTF parameters (add 1 Å to defocus range to trigger reprocessing) and see what happens.  Also try removing some frames from some of the tilt series and some tilt series completely (if these can in theory make any difference, I am not sure but that is what happens in my workflow where these effects appear). 

Edit:  re-running unblur again (inducing this can be achieved by changing the B-Factor value for instance slightly) seems to have solved the issue for me

**Reply by abartesaghi** [reply]
We'll try to reproduce the problem on our end, but glad to hear that you can already work around this issue.

**Reply by JuhaHuiskonen** [reply]
In the Table view, for which tilt is the CTF information plotted for? Is it shown for the zero tilt by default?  If there is something that would trigger showing, let's say the first or the last tilt, CTF could look weird as some of these images are blank in my dataset.

**Reply by abartesaghi** [reply]
The 2D CTF Profile in the Table view always corresponds to the 0-tilt. The 1D CTF profile shown in the Table view is estimated using an average of the 3 central images in the tilt-series (e.g., -3, 0, 3).
